tests/build.go

```diff
@@ -83,6 +89,56 @@ func testBuildRegistryExport(t *testing.T, sb integration.Sandbox) {
 	require.Equal(t, img.Layers[0]["bar"].Data, []byte("foo"))
 }
 
+func testImageIDOutput(t *testing.T, sb integration.Sandbox) {
+	dockerfile := []byte(`FROM busybox:latest`)
+
+	dir, err := tmpdir(t,
+		fstest.CreateFile("Dockerfile", dockerfile, 0600),
+	)
+	require.NoError(t, err)
+
+	targetDir := t.TempDir()
+
+	outFlag := "--output=type=docker"
+
+	if sb.Name() == "remote" {
+		// there is no Docker atm to load the image
+		outFlag += ",dest=" + targetDir + "/image.tar"
+	}
```

Comment:
:tada: tests!

I think we should probably find a way to hook into the `unsupportedFeatures` property of buildkit's `Backend` implementation, so that we could expose the driver features there, so we could rework this using `integration.CheckFeatureCompat(t, sb, driver.DockerExporter)`.

I'll open a tracking issue for this, I'll take a look after the release.